climada/hazard/trop_cyclone.py

```diff
@@ -59,11 +59,13 @@
 DEF_MAX_MEMORY_GB = 8
 """Default value of the memory limit (in GB) for windfield computations (in each thread)."""
 
-MODEL_VANG = {'H08': 0, 'H1980': 1, 'H10': 2, 'ER11': 3}
+MODEL_VANG = {'H08': 0, 'H1980': 1, 'H10': 2, 'ER11': 3, 'H10_v2': 4}
 """Enumerate different symmetric wind field models."""
 
 RHO_AIR = 1.15
 """Air density. Assumed constant, following Holland 1980."""
+RHO_AIR_SURFACE = 1.2
+"""Air density at surface level. Assumed constant, following Holland 2010"""
```

Comment:
I'm not convinced that this is a good value. There is a formula for air pressure in Holland et al. 2010:
![image](https://github.com/CLIMADA-project/climada_python/assets/57705593/b9d5ad89-d1e2-4b1d-9c54-7aa1aaba5971)
If you set SST to 28°C, you get `rho=1.167` (for ps=1005) or `rho=1.103` (for ps=950). I don't see where Holland et al. 2010 suggest to use the value 1.2. I think that it would be fine to stick to the existing constant `RHO_AIR` also for surface air.